### PR TITLE
Contact us page completed

### LINKED ISSUE #101: Create Contact Us! Dialog
Similar to the join newsletter prompt, we want to have a way for sponsors, speakers, etc. to be able to reach out!

DoD:
- There is a contact us box with Namw, Contact Reason, Additional Details, and a Send button that uses a mailto link.
- The style is according to the Figma design
- Button is primary light
- Background is primary and shadow is primary dark
- Contact Us! is H1
- The reason field has the options: Sponsorship, Speaking, Event Logistics, Media, Other

This relies on a few issues such as the drop-down box and additional details enlarged text box.

### PR DESCRIPTION
**Pull request by Ajknight121**
resolves #101 #102 #104 #106 #107 #108

### CODE REVIEW

File: src/components/ContactUs/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    <input type="text" placeholder="Name" />
+                    <div className={"drop-down"}>
+                        <div className={"arrow-down"}/>
+                        <select required value={value} onChange={handleChange}>
+                            <option value="" hidden>Reason</option>
+                            <option value={"Sponsorship"}>Interested in sponsoring</option>
+                            <option value={"Speaker"}>Interested in being a speaker</option>
+                            <option value={"Other"}>Other</option>
+                        </select>
+                    </div>
```
Comment (ChristianBingman) [inline src/components/ContactUs/index.jsx:23]: Mobile version doesn't seem to be working

File: src/index.css [hunk 2]
```diff
@@ -50,4 +50,30 @@ h5 {
...
+
+textarea[type = 'textarea'] {
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+}
+
+select{
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+}
```
Comment (ChristianBingman) [inline src/index.css:63]: The clickable portion doesn't extend to the end of the box which is confusing when there is an arrow.

File: src/components/ContactUs/index.jsx (hunk 1 above)
Comment (ChristianBingman) [reply src/components/ContactUs/index.jsx:23]: Also on safari the box with shadow is stuck to the top left and isn't centered. Probably a grid not well supported issue.

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Couple of changes, mostly minor

File: src/components/ContactUs/index.jsx (hunk 1 above)
Comment (Ajknight121) [reply src/components/ContactUs/index.jsx]: I can't check safari but I have changed the container div to flex

**Review by ChristianBingman — APPROVED** [review]
Amazing job! Thanks!

**Pull request merged**